examples/contextswitch.rs

```diff
@@ -0,0 +1,179 @@
+// Copyright 2019-2020 Twitter, Inc.
+// Licensed under the Apache License, Version 2.0
+// http://www.apache.org/licenses/LICENSE-2.0
+
+use bcc::core::BPF;
+use bcc::perf::{PerfSoftwareConfig, PerfType};
+use bcc::BccError;
+use clap::{App, Arg};
+
+use core::sync::atomic::{AtomicBool, Ordering};
+use std::sync::Arc;
+use std::{mem, ptr, thread, time};
+
+// Both consants are arbitrary
```

Comment:
what are the units?

Reply:
FREQ = Hertz
DURATION = seconds

Reply:
Can you add a comment next to both of the chosen defaults? =)